frontend/src/pages/HistoryPage.vue

```diff
@@ -93,170 +46,23 @@ export default {
-                    address: 'ул. Р.Стонз, 62',
-                    latitude: 50.486555,
-                    longitude: 30.329787,
-                    zip: null,
-                    creator_id: null,
-                    categories: {
-                        id: 10,
-                        name: 'Art Pub'
-                    },
-                    place_photo: {
-                        url: 'https://static1.squarespace.com/static/5523fa93e4b0b65a7c7d15b4/t/576050c5a3360c552edf9b4f/1465929930315/',
-                        description: 'Beer&Blues',
-                        posted_at: '2018-08-08'
-                    },
-                    reviews: {
-
-                    },
-                    places_tr: {
-                        id: 2,
-                        place_name: 'Beer&Blues',
-                        place_description: 'beer and blues',
-                        language_id: 2
-                    },
-                    cities: {
-                        id: 12,
-                        name: 'Kiev'
-                    }
-                },
-                {
-                    id: 4,
-                    ratings: {
-                        rating: 6.3
-                    },
-                    address: 'Cat str., 43',
-                    latitude: 49.817136,
-                    longitude: 24.005504,
-                    zip: null,
-                    creator_id: null,
-                    categories: {
-                        id: 10,
-                        name: 'Park'
-                    },
-                    place_photo: {
-                        url: 'https://i.ytimg.com/vi/3lqlc-ooJpE/hqdefault.jpg',
-                        description: 'Cheburek',
-                        posted_at: '2018-08-08'
-                    },
-                    reviews: [
-                        {
-                            id: 1,
-                            rate: 2
-                        },
-                        {
-                            id: 2,
-                            rate: 2
-                        },
-                        {
-                            id: 3,
-                            rate: 2
-                        }
-                    ],
-                    places_tr: {
-                        id: 2,
-                        place_name: 'Cheburek',
-                        place_description: 'Cheburek is delicious',
-                        language_id: 2
-                    },
-                    cities: {
-                        id: 10,
-                        name: 'Lviv'
-                    }
-                },
-            ]
+            visitedPlaces: null,
+            isPlacesLoaded: false,
         };
     },
     computed: {
         ...mapGetters('map', ['getMapboxToken', 'getMapboxStyle'])
     },
+    created() {
+        this.$store.dispatch('place/loadCheckInPlaces')
+            .then((response) => {
+                this.visitedPlaces = response;
```

Comment:
this is your mistake. First of all, you should put data to some module, secondary don't put data to state directly from the server. Make sure that response has data that you expect.